backend/models/face_payment.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+import (
+	"time"
+)
+
+type FacePaymentAccount struct {
+	ID				uint 			`gorm:"primaryKey; autoIncrement" json:"id"`
+	SessionID    	string  		
+	Visitor      	Visitor 		`gorm:"foreignKey:SessionID"`
+	FullName		string
+	Phone			string 			`gorm:"uniqueIndex"`
+	Pin				string
```

Comment:
Pin ini nggak int aja?
Soalnya definisinya PIN itu Personal Identification Number dan validasinya nanti harus int juga
Wdyt?

Reply:
Dibuat string karena ga bakal ada operasi matematika buat pin

Reply:
Kalo dari frontend nya "ABC123" gimana? validasinya lu buat terpisah berarti? bukan dari struct

Reply:
Iya musti dijaga bareng harusnya dari frontend juga ada validasi IsNumber. A reference: https://stackoverflow.com/questions/14171253/best-sql-type-for-a-credit-card-password

Reply:
Oalah okey